src/Components/OATHeader/OATHeader.tsx

```diff
@@ -394,6 +399,48 @@ const OATHeader = ({ elements, dispatch, state }: OATHeaderProps) => {
         onFilesUpload(acceptedFiles);
     }, [acceptedFiles]);
 
+    const onKeyDown = (e) => {
+        if (keyDown.current) {
+            // Prevent keydown from being triggered twice
+            return;
```

Comment:
Could we use this property on the event to avoid having to use a ref to hold this state? https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/repeat